src/layouts/Header/components/GuestHeader/GuestHeader.tsx

```diff
@@ -64,6 +71,7 @@ function GuestHeader() {
           )}
         </div>
         <div className="flex items-center gap-4 md:mx-4">
+          {darkMode && <ThemeToggle />}
```

Comment:
nit: convert these to ternary ? x : null

Reply:
YES!

Reply:
(please, follows our guidelines)